platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -423,172 +428,243 @@ public void setHttpHandler(final HttpHandler handler) {
     }
 
     /**
-     * Set the geographic position of the center of the map view
-     * @param position LngLat of the position to set
+     * Set the camera position of the map view
+     * @param camera CameraPosition to set
      */
-    public void setPosition(@NonNull final LngLat position) {
-        checkPointer(mapPointer);
-        nativeSetPosition(mapPointer, position.longitude, position.latitude);
+    public void setCameraPosition(@NonNull final CameraPosition camera) {
```

Comment:
Theoretically we can get rid of the methods that move the camera with a `CameraPosition` - using a `CameraUpdate` covers a superset of their functionality.

Pros:
 - Makes the interface simpler and easier to understand
 - Less code, easier to maintain

Cons:
 - Creates more new objects than strictly necessary, could worsen GC